test/kixi/integration/ses_test.clj

```diff
@@ -181,6 +183,7 @@
 
 (def test-group-mail
   {:kixi.mailer/destination {:kixi.mailer.destination/to-groups #{"c645d47d-1236-4dda-a16f-2d33941b5993" ;; AW
+                                                                  "0ace7b64-4f2a-4665-8784-b44ff7be63db" ;; 'The Toms'
```

Comment:
Would having a group distribution like email be a more prudent move in case you are both unavailable?

Reply:
It's not really interesting who the email goes to - just that it gets sent. I picked these group IDs because one's a user and one's a group.